deploy/kustomization.yaml

```diff
@@ -19,4 +19,4 @@ kind: Kustomization
 images:
```

Comment:
need to remove this commit.  I have to fix the way we use kustomize so that that this file stops getting modified.